### DISCUSSION TITLE
Automatically create new constant when dragging and dropping .tscn files into the script editor

### DISCUSSION THREAD

**Discussion by nubels** [godotengine/godot-proposals#7841] [Editor]
It would be a nice UX improvement if you could drag and drop .tscn files into the script editor and have Godot automatically create a constant for you similar to how it works when dragging and dropping nodes while holding control. This way, you would not have to declare the constant.

![Godot_v4 1 1-stable_win64_dydfVo6U7a](https://github.com/godotengine/godot-proposals/assets/81755541/27dfdc0d-5c1b-41f6-aeab-082228ef20d7)

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/7673.
